Code/Framework/AzFramework/AzFramework/Script/ScriptRemoteDebuggingConstants.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Name/Name.h>
+
+namespace AzFramework
+{
+    static const AZ::Name LuaToolsName = AZ::Name::FromStringLiteral("LuaRemoteTools");
```

Comment:
it feels weird having these here vs. with the LuaIDE stuff, but maybe it's better than importing something from Tools/LuaIDE into `Code/Framework/AzFramework/AzFramework/Application/Application.cpp`. Is the Idea that we'd add more here as non-Lua remote debugging tools (potentially) get added?

Reply:
The problem is Tools/LuaIDE is pretty limited with respect to where it's pulled in . This is the most appropriate place for this that Application.cpp can pull in.

Reply:
Well this is more of a limitation that the Lua integration is directly within AzFramework instead of being with a Gem.
If we move the Lua integration from AzFramework into its own gem, then all of the RemoteTools registration and files within the `AzFramework/Script` folder will be moved over to that gem.